solr/contrib/ltr/example/solrconfig.xml

```diff
@@ -867,7 +867,9 @@
 
 
   <!-- Component that hooks up managed resources for features and models -->
-  <searchComponent name="ltrComponent" class="org.apache.solr.ltr.ranking.LTRComponent"/>
+  <searchComponent name="ltrComponent" class="org.apache.solr.ltr.ranking.LTRComponent">
+     <int name="LTRThreads">10</int>
```

Comment:
see if we can move this param into the ltrqparserplugin config since we are looking to delete the ltrcomponent